test/conformance/channel_addressable_resolver_cluster_role_test.go

```diff
@@ -25,6 +25,7 @@ import (
 	testlib "knative.dev/eventing/test/lib"
 )
 
+// Deprecated
```

Comment:
For all these deprecated methods, what should be used instead? Maybe add a comment here what to do instead?

Reply:
I am marking them to delete later so I can find them and track what I have replaced

Reply:
Given that, what would you suggest to add in the comment? When we are done with all this, that directory is going to be deleted.

Reply:
Talked offline, these are going to be removed in the future and only using rekt stuff.